scripts/events_module/relationship/group_events.py

```diff
@@ -553,29 +533,29 @@ def influence_general_relationship(
         dictionary = chosen_interaction.general_reaction
 
         # set the amount
-        romantic = 0
-        platonic = 0
-        dislike = 0
-        admiration = 0
-        comfortable = 0
-        jealousy = 0
+        romance = 0
+        like = 0
+        respect = 0
         trust = 0
-        if "romantic" in dictionary and dictionary["romantic"] != "neutral":
-            romantic = amount if dictionary["romantic"] == "increase" else amount * -1
-        if "platonic" in dictionary and dictionary["platonic"] != "neutral":
-            platonic = amount if dictionary["platonic"] == "increase" else amount * -1
-        if "dislike" in dictionary and dictionary["dislike"] != "neutral":
-            platonic = amount if dictionary["dislike"] == "increase" else amount * -1
-        if "admiration" in dictionary and dictionary["admiration"] != "neutral":
-            platonic = amount if dictionary["admiration"] == "increase" else amount * -1
-        if "comfortable" in dictionary and dictionary["comfortable"] != "neutral":
-            platonic = (
-                amount if dictionary["comfortable"] == "increase" else amount * -1
+        comfort = 0
+
+        if RelValue.ROMANCE in dictionary:
+            romance = (
+                amount if dictionary[RelValue.ROMANCE] == "increase" else amount * -1
             )
-        if "jealousy" in dictionary and dictionary["jealousy"] != "neutral":
-            platonic = amount if dictionary["jealousy"] == "increase" else amount * -1
-        if "trust" in dictionary and dictionary["trust"] != "neutral":
-            platonic = amount if dictionary["trust"] == "increase" else amount * -1
+        if RelValue.LIKE in dictionary:
+            like = amount if dictionary[RelValue.LIKE] == "increase" else amount * -1
+        if RelValue.RESPECT in dictionary:
+            respect = (
+                amount if dictionary[RelValue.RESPECT] == "increase" else amount * -1
+            )
+        if RelValue.TRUST in dictionary:
+            trust = amount if dictionary[RelValue.TRUST] == "increase" else amount * -1
+        if RelValue.COMFORT in dictionary:
+            comfort = (
+                amount if dictionary[RelValue.COMFORT] == "increase" else amount * -1
+            )
```

Comment:
```suggestion
        amount_dict = {
            RelValue.ROMANCE: 0,
            RelValue.LIKE: 0,
            RelValue.RESPECT: 0,
            RelValue.TRUST: 0,
            RelValue.COMFORT: 0
        }
        for key in amount_dict.keys():
            if key in dictionary:
                amount_dict[key] = amount if dictionary[key] == "increase" else amount * -1
```
this comment is supposed to be on lines `536-557` but github is being a butt and not loading so here we are